sql/telemetry_derived/clients_daily_active_hours_by_top_city_aggregates_v1/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: Clients daily active hours by top cities aggregates
```

Comment:
Those query names are getting pretty long :/

Reply:
Unless we expect these to have direct application outside of this anomaly detection pipeline, we could name these all with an `anomdtct_` prefix to match the library name. So this could be maybe `anmdtct_top_city_aggregates_v1`.

Reply:
These tables won't be necessary if the models get cached in BigQuery and then used for doing predictions on new data. Which is probably the most efficient way to go.